examples/wrapper_example_1.py

```diff
@@ -49,27 +38,24 @@ def create_plot(scores):
 
     # Comparable with examples/example_2.py
     model = SupRBWrapper(print_config=True,
-                         
-                        ## RULE GENERATION ##
-                        rule_generation=ES1xLambda(), 
-                        rule_generation__n_iter=10, 
-                        rule_generation__lmbda=16, 
-                        rule_generation__operator='+', 
-                        rule_generation__delay=150, 
-                        rule_generation__random_state=random_state, 
-                        rule_generation__n_jobs=1, 
-
-                        ## SOLUTION COMPOSITION ##
-                        solution_composition=GeneticAlgorithm(),
-                        solution_composition__n_iter=32,
-                        solution_composition__population_size=32,
-                        solution_composition__elitist_ratio=0.2,
-                        solution_composition__random_state=random_state,
-                        solution_composition__n_jobs=1)
+
+                         ## RULE GENERATION ##
+                         rule_generation=ES1xLambda(),
+                         rule_generation__n_iter=10,
+                         rule_generation__lmbda=16,
+                         rule_generation__operator='+',
+                         rule_generation__delay=150,
+                         rule_generation__random_state=random_state,
+                         rule_generation__n_jobs=1,
+
+                         ## SOLUTION COMPOSITION ##
+                         solution_composition=GeneticAlgorithm(),
+                         solution_composition__n_iter=32,
+                         solution_composition__population_size=32,
+                         solution_composition__elitist_ratio=0.2,
+                         solution_composition__random_state=random_state,
+                         solution_composition__n_jobs=1)
 
     scores = cross_validate(model, X, y, cv=4, n_jobs=1, verbose=10,
```

Comment:
right now, the script just ends without further outputs. Please print the scores in a sensible manner for all scripts

Reply:
Added `log_scores(scores)` which gives this kind of output:
![{3660BC9A-B1B5-4CAD-BA89-E9A03D5D241F}](https://github.com/user-attachments/assets/14af13c4-a72d-4c0f-aa59-404e6154708d)

Reply:
is it intended that the mixing models dont resolve to text but the object address is printed?